### PR TITLE
Use MGLEcho to provide back pressure for Mozart

### PR DESCRIPTION
**Pull request by abarth**
Related to https://github.com/domokit/mojo/issues/684

### CODE REVIEW

**Comment by abarth** [comment]
@jamesr @j9brown @apwilson

**Comment by j9brown** [comment]
lgtm, this kind of mechanism should help a lot

File: sky/shell/gpu/mojo/rasterizer_mojo.cc [hunk 1]
```diff
@@ -12,6 +14,16 @@
 
 namespace sky {
 namespace shell {
+namespace {
+
+void DidEcho(void* context) {
```
Comment (krisgiesing) [inline sky/shell/gpu/mojo/rasterizer_mojo.cc]: Suggest adding an explanatory comment

Comment (abarth) [inline sky/shell/gpu/mojo/rasterizer_mojo.cc]: Mission accomplished

**Pull request merged**